### DISCUSSION TITLE
USPS Camera image missing

### DISCUSSION THREAD

**Discussion by GizmoGoody** [moralmunky/Home-Assistant-Mail-And-Packages#716] [Q&A] [answered]
~A few weeks ago, my USPS Camera stopped updating. This may have been around the time I installed the latest HA update, but cannot be sure. I've uninstalled and reinstalled the integration, restarted HA, triple-checked my configuration, and cleared my cache but still no image. When I have no mail from informed delivery, I do get the No Mail image, but once that sensor updates for the day, I have no gif for camera.mail.usps.camera.

Previously, I did not have to use either option presented here - https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/wiki/USPS-Informed-Delivery-Image to get the image to display directly from the Entities page or when used in a dashboard (click for more-info). But, the sensor.mail_image_url state is currently "Unknown" and when I navigate to the location provided by sensor.mail_image_system_path, the file is not there. Actually, there are 0 .gif files in that folder (I'm displaying hidden items).

Anyone else having this issue? What should I check next?

**Comment by firstof9** [comment, marked as answer]
Use the latest beta version.

**Reply by GizmoGoody** [reply]
@firstof9 0.3.5-b13 fixed my issue. Thank you.
